### ISSUE TITLE
Reggression tests

### ISSUE THREAD

**Issue by ajnavarro** [src-d/gitbase#254]

**Comment by jfontan** [comment]
Blocked while final tables are not there to create test queries.

**Comment by ajnavarro** [comment]
Not blocked anymore.

**Comment by erizocosmico** [comment]
This is done, right? https://github.com/src-d/regression-gitbase

**Comment by ajnavarro** [comment]
It is an ongoing task yet.

**Comment by ajnavarro** [comment]
- We changed a lot the supported statements after the regression test started, so we stopped it.
- After that, we added indexes, and we started implementing a way to import and export indexes. We changed that too, so we stopped again.

Right now, after a huge speed improvement on index creation, maybe does not make sense implement index import/export, maybe is enough just executing the "CREATE INDEX" statement.

We should support the execution of, at least, gitbase versions >=0.13.0.

We should be able to test any kind of queries, even index creation ones.
